src/UseCases/ExtractAddress/CSVFormatter.php

```diff
@@ -0,0 +1,17 @@
+<?php
+declare( strict_types = 1 );
+
+namespace WMDE\OtrsExtractAddress\UseCases\ExtractAddress;
+
+trait CSVFormatter {
```

Comment:
Why trait and not composition? If you don't want to inject the thing I'd still make a class and just instantiate it where it is used.